modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -66,6 +71,17 @@ def allow_processing?
       Flipper.enabled? :claims_hourly_slack_error_report_enabled
     end
 
+    def get_filtered_unique_errors
+      unique_errors = get_unique_errors # unique by transaction_id
+      filtered_errors = []
+
+      unique_errors.each do |ue|
+        filtered_errors << ue[:id] unless NO_INVESTIGATION_ERROR_TEXT.any? { |text| ue[:evss_response]&.include?(text) }
+      end
+
+      filtered_errors
```

Comment:
nit: `filtered_error_ids`? 🤷

Reply:
[e0471a6](https://github.com/department-of-veterans-affairs/vets-api/pull/19273/commits/e0471a63dd2835b59ffec66c5f12b6e3aef11f34)